jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/PullBaseImageStep.java

```diff
@@ -95,47 +95,48 @@ public ImagesAndRegistryClient call()
           LayerCountMismatchException, BadContainerConfigurationFormatException,
           CacheCorruptedException, CredentialRetrievalException {
     EventHandlers eventHandlers = buildContext.getEventHandlers();
-    // Skip this step if this is a scratch image
-    ImageReference imageReference = buildContext.getBaseImageConfiguration().getImage();
-    if (imageReference.isScratch()) {
-      Set<Platform> platforms = buildContext.getContainerConfiguration().getPlatforms();
-      Verify.verify(!platforms.isEmpty());
-
-      eventHandlers.dispatch(LogEvent.progress("Getting scratch base image..."));
-      ImmutableList.Builder<Image> images = ImmutableList.builder();
-      for (Platform platform : platforms) {
-        Image.Builder imageBuilder = Image.builder(buildContext.getTargetFormat());
-        imageBuilder.setArchitecture(platform.getArchitecture()).setOs(platform.getOs());
-        images.add(imageBuilder.build());
+    try (ProgressEventDispatcher progressEventDispatcher =
+            progressEventDispatcherFactory.create("pulling base image manifest", 2);
```

Comment:
BTW, looks like the current allocation units (2) isn't correct, but I'm not addressing this here.